pallets/nfts/src/lib.rs

```diff
@@ -1345,6 +1452,55 @@ pub mod pallet {
 			Self::do_cancel_approval(maybe_check_origin, collection, item, delegate)
 		}
 
+		/// Cancel a collection approval.
+		///
+		/// Origin must be Signed.
+		///
+		/// Arguments:
+		/// - `collection`: The collection whose approval will be cancelled.
+		/// - `delegate`: The account that is going to loose their approval.
+		///
+		/// Emits `ApprovalCancelled` on success.
+		///
+		/// Weight: `O(1)`
+		#[pallet::call_index(41)]
+		#[pallet::weight(T::WeightInfo::cancel_collection_approval())]
+		pub fn cancel_collection_approval(
+			origin: OriginFor<T>,
+			collection: T::CollectionId,
+			delegate: AccountIdLookupOf<T>,
+		) -> DispatchResult {
+			let origin = ensure_signed(origin)?;
+			let delegate = T::Lookup::lookup(delegate)?;
+			Self::do_cancel_collection_approval(origin, collection, delegate)
+		}
+
+		/// Force-cancel a collection approval granted by `owner` account.
+		///
+		/// Origin must be `ForceOrigin`.
+		///
+		/// Arguments:
+		/// - `owner`: The account cancelling approval for delegated transfer.
+		/// - `collection`: The collection of whose approval will be cancelled.
+		/// - `delegate`: The account that is going to loose their approval.
```

Comment:
```suggestion
		/// - `delegate`: The account that is going to lose their approval.
```
Please check all instances of 'loose' which should be 'lose'